app/src/main/res/values/strings.xml

```diff
@@ -232,13 +232,13 @@
     <string name="unsupport_magisk_msg">This version of the app does not support Magisk version lower than %1$s.\n\nThe app will behave as if no Magisk is installed, please upgrade Magisk as soon as possible.</string>
     <string name="unsupport_general_title">Abnormal State</string>
     <string name="unsupport_system_app_msg">Running this app as a system app is not supported. Please revert the app to a user app.</string>
-    <string name="unsupport_other_su_msg">An \"su\" command that does not belong to Magisk is detected. Please remove the other unsupported su.</string>
+    <string name="unsupport_other_su_msg">Please remove the app that just tried to handle a \"su\" command and handle all requests through Magisk instead.</string>
     <string name="unsupport_external_storage_msg">Magisk is installed to external storage. Please move the app to internal storage.</string>
-    <string name="unsupport_nonroot_stub_msg">The app cannot continue to work in the hidden state as root was lost. Please restore it back to the original APK.</string>
+    <string name="unsupport_nonroot_stub_msg">The app cannot stay hidden as root was lost. Please restore it back to the original APK.</string>
     <string name="unsupport_nonroot_stub_title">@string/settings_restore_app_title</string>
     <string name="external_rw_permission_denied">Grant storage permission to enable this functionality</string>
     <string name="add_shortcut_title">Add shortcut to home screen</string>
     <string name="add_shortcut_msg">After hiding this app, its name and icon might become difficult to recognize. Do you want to add a pretty shortcut to the home screen?</string>
-    <string name="app_not_found">No application found to handle this action</string>
```

Comment:
Not consistent with "app" used elsewhere.

Reply:
It's consistent with the other use of "No application" and it's perfect English.

Reply:
I find your application not always applicable.

Reply:
Your proposed change is more awkward.

Reply:
OK